test/xref2/github_issue_447.t/a.mli

```diff
@@ -0,0 +1,11 @@
+type u = Foo
+
+(** {!constructor-Foo} {!u.constructor-Foo} *)
+
+module M : sig
+  type t = Foo
+end
+
+(** {!M.constructor-Foo}
```

Comment:
It could be worth checking for `M.Foo` and `M.t.Foo`, which works fine in my testing.

Reply:
Done!